Consul Secure ACL Vault scenario (#355)
hashicorp/katakoda: consul-secure-acl-vault-hashiconf/step0.md

The environment consists of a Vault server and three Consul servers.

The agents are running as Docker containers but you can consider them as Virtual
Machines each containing all the binaries necessary for the environment setup.

> The startup should end with a list of environment variables that will help you
> setup your terminal environment.

Reviewer: ```suggestion
The provision will output a list of environment variables in your terminal that will help you
setup your terminal environment.
```
I assume you are referring to these:
export CONSUL_HTTP_ADDR=https://172.19.0.6:443
export CONSUL_HTTP_TOKEN=852f2332-ebba-d5af-ae44-53d964aeda17
export CONSUL_HTTP_SSL=true
export CONSUL_CACERT=./assets/secrets/consul-agent-ca.pem
export CONSUL_HTTP_SSL_VERIFY=false

This step feels a little confusing in terms of flow since the previous thing we are looking at is "Complete!  Move on to the next step.", then in this step we are looking above that and executing `source ./assets/secrets/PRODUCT_env.conf`, which feels like it could be under the hood or explained a bit more as to what they do.

Also.. switched from "startup" to "provision" to stay consistent with initial language.
